pkg/rbd/rbd_util.go

```diff
@@ -126,7 +145,7 @@ func getMon(pOpts *rbdVolume, credentials map[string]string) (string, error) {
 	return mon, nil
 }
 
-// CreateImage creates a new ceph image with provision and volume options.
+// createRBDImage creates a new ceph image with provision and volume options.
 func createRBDImage(pOpts *rbdVolume, volSz int, adminID string, credentials map[string]string) error {
```

Comment:
Nit: comment applies throughout: why add ```RBD``` to all these method names? They are inside the ```rbd``` package.

Reply:
Truthfully all names were 'extended' from the existing function name createRBDExists.


Comment does make sense, but somewhere is also jarring (the RBD in there provide a sense of comfort that the RBD function is being called). Will relook and see what would be the right naming for these.